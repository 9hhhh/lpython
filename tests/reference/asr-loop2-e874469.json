{
    "basename": "asr-loop2-e874469",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/loop2.py",
    "infile_hash": "ca62d52ec7887e3a455cec87a0e49102e94e96db925f6c37edb792e7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-loop2-e874469.stdout",
    "stdout_hash": "03f169279ca3496703b5a91f94c46bb5f0b555329f9a4bf02b2939a2",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}