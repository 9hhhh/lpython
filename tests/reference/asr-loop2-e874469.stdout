(TranslationUnit (SymbolTable 1 {test_for: (Subroutine (SymbolTable 2 {i: (Variable 2 i Local () () Default (Integer 4 []) Source Public Required .false.)}) test_for [] [(DoLoop ((Var 2 i) (ConstantInteger 0 (Integer 4 [])) (ConstantInteger 10 (Integer 4 [])) (ConstantInteger 1 (Integer 4 []))) [(If (Compare (Var 2 i) Eq (ConstantInteger 0 (Integer 4 [])) (Logical 4 []) () ()) [(Cycle)] []) (If (Compare (Var 2 i) Gt (ConstantInteger 5 (Integer 4 [])) (Logical 4 []) () ()) [(Exit)] [])])] Source Public Implementation () .false. .false.)}) [])
